src/main/java/frc/robot/CommandFactory.java

```diff
@@ -17,28 +27,50 @@ public class CommandFactory {
     private final CoralIntake coralIntake;
     private final CoralShooter coralShooter;
     private final Elevator elevator;
-    private final Vision vision; 
+    private final Vision vision;
     private final AlgaeShooter algaeShooter;
-
+    private final CommandSwerveDrivetrain drivetrain;
+    private final XboxController driverCont;    
     // ↓ constructor ↓ //
-public CommandFactory(
-    Catapult catapult,
-    CoralIntake coralIntake,
-    CoralShooter coralShooter,
-    Elevator elevator,
-    Vision vision,
-    AlgaeShooter algaeShooter
-) {
-    this.catapult = catapult;
-    this.coralIntake = coralIntake;
-    this.coralShooter = coralShooter;
-    this.elevator = elevator; 
-    this.vision = vision;
-    this.algaeShooter = algaeShooter;
-}
+    public CommandFactory(
+        Catapult catapult,
+        CoralIntake coralIntake,
+        CoralShooter coralShooter,
+        Elevator elevator,
+        Vision vision,
+        AlgaeShooter algaeShooter,
+        CommandSwerveDrivetrain driveTrain,
+        XboxController driverCont
+    ) 
+    
+    {
+        this.catapult = catapult;
+        this.coralIntake = coralIntake;
+        this.coralShooter = coralShooter;
+        this.elevator = elevator;
+        this.vision = vision;
+        this.algaeShooter = algaeShooter;
+        this.drivetrain = driveTrain;
+        this.driverCont = driverCont;
+    }
 
-public Command setElevatorHeight(double height) {
-    return elevator.setElevatorHeight(height);
-}
+    /*
+     * height is in motor rotations!
+     */
+    public Command setElevatorHeight(double height) {
+        return elevator.setElevatorHeight(height);
+    }
+
+    public Command AlignWithLimelight(double goalTY, double goalTX, int pipeline) {
+        return //vision.waitUntilTargetTxTy(goalTX, goalTY).alongWith(drivetrain.waitUntilDrivetrainAtHeadingSetpoint())
+            (new AlignWithLimelight(vision, drivetrain, goalTY, goalTX,
+                        WoodBotDriveTrain.kSpeedAt12Volts.in(MetersPerSecond), pipeline, driverCont)); // no more timeout
+    }
 
+    public Command allignToReefWoodbotLeft(){
+        return new SequentialCommandGroup(
+            new SnapDrivebaseToAngle(drivetrain),
+            new AlignWithLimelight(vision, drivetrain, -12.64, -11.16, 1, 0, driverCont)
```

Comment:
This looks like we could remove this because we aren't using this method and are instead setting the set points in robot container